docs/architecture/deployment/installation-guide.md

```diff
@@ -36,38 +36,105 @@ To install Prophecy, the following infrastructure requirements must be satisfied
+| `global.prophecy`                          | -                        |                                                                                                                                                                                                                                  |                                     |
+| `global.prophecy.domainValue`              | string                   | domain value used for cookies. Used to configure multiple URLs.                                                                                                                                                                  | `""`                                |
+| `global.prophecy.envSecret`                | -                        | Is configred when Prophecy credentials are fetched as a kubernetes secret                                                                                                                                                        |                                     |
+| `global.prophecy.envSecret.create`         | bool                     | Set to true if the secret is to be created. With `false` the secret is expected to be created by the customer.                                                                                                                   | `false`                             |
+| `global.prophecy.envSecret.data`           | map[string]string        | Provide the values you wish to override in Prophecy secret ENV variables. Passed in the format of `METADATA_DB_PASSWORD: dummyPassword123`.                                                                                      |                                     |
+| `global.prophecy.envSecret.name`           | string                   | Name of the Kubernetes secret to be used.                                                                                                                                                                                        | `env-secrets`                       |
+| `global.prophecy.imagePullSecret`          | string                   | Name of the Kubernetes secret which contains the image pull credentials to the container registry.                                                                                                                               | `""`                                |
+| `global.prophecy.rootUrl`\*                | string                   | Root URL host used to identify the DNS traffic. You may provider a list of comma separated urls.                                                                                                                                 | `""`                                |
```

Comment:
Description: Root URL where the Prophecy deployment will be hosted. You may provider a list of comma separated urls if you wish to work with multiple urls. Note that the first url in the provided list will be considered as the primary url.